src/libsyntax/attr/mod.rs

```diff
@@ -340,7 +340,10 @@ impl Attribute {
                 DUMMY_SP,
             );
             f(&Attribute {
-                item: AttrItem { path: meta.path, tokens: meta.kind.tokens(meta.span) },
+                item: AttrItem {
+                    path: meta.path,
+                    tokens: TokenStream::new(meta.kind.token_trees_and_joints(meta.span)),
```

Comment:
Could you keep the `tokens` methods as wrappers around new `token_trees_and_joints`?
They are still used in multiple places.